DaemonCommander/ResponseClasses.cs

```diff
@@ -24,10 +24,20 @@ public class StartResult
 
     public class ClusterConfig
     {
+        public string ClusterCACert { get; set; }

```

Comment:
This commit is doing 3 different things, without explicit references to the initial bug, so it's very hard to know which of these 3 changes is the root cause of the bug and which ones are code/robustness improvements, or if all 3 are needed to fix the bug. Having an explanation of what exactly was causing the bug is likely to be useful if a similar bug occurs in a few months.

Reply:
Perhaps 2 commits, as the Response just adds additional expected fields.

Reply:
I split the commits.